### ISSUE TITLE
`Vagrant up` deletes vagrant.exe on Windows 7

### ISSUE THREAD

**Issue by refactormyself** [hashicorp/vagrant#11434]
### Vagrant version
Vagrant 2.2.7
### Host operating system
Windows 7 64-bits

### Guest operating system
ubuntu/focal64 and ubuntu/precise64 
### Vagrantfile
default from running `vagrant init ubuntu/focal64`

### Debug output
Provide a link to a GitHub Gist containing the complete debug output:
https://www.vagrantup.com/docs/other/debugging.html. The debug output should
be very long. Do NOT paste the debug output in the issue, just paste the
link to the Gist.

### Expected behavior
After `vagrant up`, I should be able to do `vagrant ssh`

### Actual behavior
Got this :
```
vagrant : The term 'vagrant' is not recognized as the name of a cmdlet, function,
script file, or operable program. Check the spelling of the name, or if a path was
included, verify that the path is correct and try again.
At line:1 char:1
+ vagrant ssh
+ ~~~~~~~
    + CategoryInfo          : ObjectNotFound: (vagrant:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
Basically the command `vagrant up` deletes `C:\HashiCorp\Vagrant\bin\vagrant.exe`. Probably out of anger!
### Steps to reproduce
1. `vagrant init ubuntu/focal64`
2. `vagrant up`
3. If there is still peace, try `vagrant ssh`
4. Try another box, I earlier thought it was my install but after a fresh install it happens again.
5. Please note that I install Ruby with `choco install ruby`, but why should cause any anger?!

### References
I think it may be related to #11382

**Comment by jbonhag** [comment]
Hi there,

Thanks for reporting this issue! Is it possible that you have anti-virus software running?  If so, the anti-virus may be quarantining the `vagrant.exe` file after the `vagrant up` step.

**Comment by refactormyself** [comment]
Wow, you are so right. This is shameful, it's my last week on Windows anyways, I am done.
![image](https://user-images.githubusercontent.com/17991837/76121824-dd703a00-5ff4-11ea-8c8a-55ab3435cc07.png)

**Comment by jbonhag** [comment]
No problem, glad it's figured out. Cheers and have a great day! 😄
